base-isa.md

```diff
@@ -99,13 +99,12 @@ Undefined control registers are reserved and reading from or writing to them is
 
 | `CRN`  | NAME       | Description                                                                                                              | Comment |
 |--------|------------|--------------------------------------------------------------------------------------------------------------------------|---------|
-| `0000` | `CPUID`    | Reading from this control register puts the available CPU extensions in the destination register. Writing to it is a NOP | (1)     |
-| `0001` | `EXTEN`    | This control register specifies which available extensions are enabled or disabled.                                      | (2)     |
-| `0010` | `FEAT`     | Reading from this control register puts the available CPU features in the destination register. Writing to it is a NOP   | (3)     |
+| `0000` | `CPUID1`   | Reading from this control register puts the available CPU extensions in the destination register. Writing to it is a NOP | (1)     |
+| `0001` | `CPUID2`   | Reading from this control register puts the available CPU extensions in the destination register. Writing to it is a NOP | (1)     |
```

Comment:
This sounds like it's a full duplicate of `CPUID1`.

Reply:
I wasn't sure how to properly word it, but it's a second CPUID CR which has a different set of extensions specified. The main purpose is to allow us to have more extensions which don't depend on 32/64 bit operations to detect their presence or lack thereof. It also gives us a second set of easy to access extensions which would be useful to detect.